pixeltable/exec/cache_prefetch_node.py

```diff
@@ -255,7 +255,9 @@ def __fetch_url(self, url: str) -> tuple[Optional[Path], Optional[Exception]]:
                     data = resp.read()
                     f.write(data)
             else:
-                raise AssertionError(f'Unsupported URL scheme: {parsed.scheme}')
+                raise AssertionError(
+                    f'INTERNAL ERROR: __fetch_url encountered an unsupported URL scheme: {parsed.scheme}'
```

Comment:
This isn't really helpful since the stack trace will show us that the assertion fired inside `__fetch_url` (so the error message adds no new information)